src/dios-egraphs/Cargo.toml

```diff
@@ -6,14 +6,23 @@ edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[lib]
+name = "llvmlib"
+crate-type = ["rlib", "cdylib"]
+
 [dependencies]
 egg =  "0.6.0"
 assert_approx_eq = "1.1.0"
 itertools = "0.9.0"
 clap = "2.33.2"
 lexpr = "0.2.5"
+llvm-sys = { version = "110", features = ["no-llvm-linking"] }
```

Comment:
This does seem to make "normal" Diospyros also needlessly require LLVM. It would be cool to separate things, probably into three Cargo projects:

- core functionality library
- CLI "frontend"
- LLVM pass

...but IMO that can happen in a subsequent PR; it's better to get this basic stuff merged first.